### DISCUSSION TITLE
Zemismart ZM25EL -- motors showing different manufacture #s + one not showing battery levels?

### DISCUSSION THREAD

**Discussion by Ltek** [Koenkk/zigbee2mqtt#18893] [Devices]
I bought 6 Zemismart ZM25EL... they read as two different manufactures, with different features?

1 reads as TS0601_cover_3 (https://www.zigbee2mqtt.io/devices/TS0601_cover_3.html#tuya-ts0601_cover_3) 
... manufacture _TZE200_68nvbio9
... this reports battery level and several other features the other does not

5 read as TS0601_cover_1 (https://www.zigbee2mqtt.io/devices/TS0601_cover_1.html)
... manufacture _TZE200_pw7mji0l

Is this correct? Is there a way to get cover_1 to report battery or get the other features?

thx!

**Comment by kenmaca** [comment]
I'm seeing the same issue with my new ZM25EL's. Were you able to fix this?

**Reply by Ltek** [reply]
nope... looks like a bug, or gap, in Z2M
